### DISCUSSION TITLE
Suggestion fir pictures in pdf

### DISCUSSION THREAD

**Discussion by gurvanmeunier** [jzillmann/pdf-to-markdown#67] [Ideas]
Why not extract picture and hard code them in base64 directly in the md final file ?

**Comment by darkcheftar** [comment]
It sounds like a good idea, May be good behind a flag

**Comment by gurvanmeunier** [comment]
```
![alt][1]  //where the picture have to be displayed e picture need to be displayed 

 [1]:data:image/png;base64,iVBORw0KGg "title" // at the end of the file
```

**Reply by jzillmann** [reply]
Yeah, extracting pictures would be cool. Didn't know you can base64 encode them...

**Comment by gurvanmeunier** [comment]
the only inconvenience is the amount of code in the md file

**Reply by darkcheftar** [reply]
Agreed, I once tried to create notes with md and hosted it, you can see it [here](https://darkcheftar.github.io/SecondYear_EvenSem/Subjects/java/10.JDBC.html) and for images to not to host seperately, I have encoded them in base64.
